staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -117,42 +167,48 @@
       <parent link="${prefix}base_link"/>
       <child link="${prefix}link_1"/>
       <axis xyz="0 0 1"/>
-      <limit lower="-2.967060" upper="2.967060" effort="0" velocity="2.88" />
+      <limit lower="-2.967060" upper="2.967060" effort="692.0" velocity="2.88" />
```

Comment:
I have an older version of the Staubli instruction manual than the one referred to in `package.xml` (D28086504A), though I expect torque limits to be the same. In version D28075404C, the values can be found in a table in Section 2.6.2 (Torque Limits).

Note that I would allow maximum torque on axis 6, thus setting `effort` to 29 on axis 5, rather than a feasible 58 when torque on axis 6 = 0 (see superscripts (1) and (2) for details).